app/cli/cmd/policy_develop_lint.go

```diff
@@ -55,11 +55,7 @@ func newPolicyDevelopLintCmd() *cobra.Command {
 				return nil
 			}
 
-			// Print all validation errors
-			for _, err := range result.Errors {
-				logger.Error().Msg(err)
-			}
-			return fmt.Errorf("policy validation failed with %d issues", len(result.Errors))
+			return encodeResult(result)
```

Comment:
what's the output when we do not have any violations? Do we return an empty violations array?

Reply:
awesome, please merge :)